proto/signer.pb.go

```diff
@@ -1,12 +1,12 @@
 // Code generated by protoc-gen-go.
-// source: proto/signer.proto
+// source: signer.proto
```

Comment:
I think it's relative from where you run the `protoc` command.  If you run `make protos` from the main repo directory you get `proto/signer.proto`